cli/src/config-schema.json

```diff
@@ -164,7 +164,8 @@
                     "description": "Conflict marker style to use when materializing conflicts in the working copy",
                     "enum": [
                         "diff",
-                        "snapshot"
+                        "snapshot",
```

Comment:
(For discussion) For the snapshot style, we could also do something more similar to `git` style, something like

```
<<<<<<< Side 1 (Conflict 1 of 1 begins)
fn example(word: String) {
    println!("word is {word}");
||||||| Base
fn example(w: String) {
    println!("word is {w}");
======= Side 2
fn example(w: &str) {
    println!("word is {w}");
>>>>>>> Conflict 1 of 1 ends
}
```

One downside of this is that, beyond the number of sides (which could be larger than 2 here) and the placement of comments, there would be another subtle difference between this style and git's style once we fix https://github.com/martinvonz/jj/issues/3975, since the separators could get longer.

If we go with this, I'd go with a different name, e.g. "git-extended".

Reply:
I personally think it would be nice to keep the "diff" and "snapshot" styles as similar as possible, because I feel like it gives a nice sense of consistency. In my mind, "diff" and "snapshot" would be the preferred Jujutsu-style conflict markers, and any other conflict marker styles would be added mainly for compatibility with other tools.

On the other hand, similarity to Git could be good for new users because it's less for them to learn.

Reply:
I agree that we should keep them as similar as possible.

Reply:
Do you mean `snapshot` and the current jj style or `snapshot` and `git`? I'm guessing it's the first one, and I think I'm also leaning towards it, though I'm not sure.

Reply:
Ah, yes, that's what I mean.

I think new tools should not care too much about being similar to old tools. That just benefits existing users of the old tool. It should not be at the expense of what we think is best for new users. I think we should at least hope to have more future users than migrating users in the long run (though that's a pretty ambitious when we're talking about a tool with ~98% market or whatever it is Git has).